db/models/index.js

```diff
@@ -15,12 +15,15 @@ module.exports = {
 
   fillData: {
     post: (callback) => {
-      db.con.query('SELECT COUNT(*) FROM Project', (err, data) => {
-        if (err) {
-          callback(err, null);
-        } else {
-          log.info(data);
-        }
+      db.con.connect((connectionError) => {
```

Comment:
I would recommend you to invoke the connect function once in db/index.js and leave it open for all the interactions.

Reply:
@despinoz My tests were failing because I was leaving the connections open. I decided to do it this way so that I can close the connection after using it so as to not leave any open connections to the database.

Reply:
Take a look on Michael code or mine about that, on a certain moment your component will need to work with the connection open. Consider that for future commits.